### PR TITLE
contributing.md added

### PR DESCRIPTION
**Pull request by alganthe**
Added a CONTRIBUTING.md [as described here](https://github.com/blog/1184-contributing-guidelines)

### CODE REVIEW

**Comment by bux578** [comment]
Nice! But this file should also include the steps in (or a link to) 
* http://ace3mod.com/wiki/development/setting-up-the-development-environment.html
* http://ace3mod.com/wiki/development/coding-guidelines.html

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+# Feature Requests
+
+## Background
+ACE2, AGM and CSE had a lot of features implemented or planned. All of them are or have been evaluated for inclusion in ACE3 itself, and we'd like to port the majority of them eventually. However, due to time constraints, we have managed to finish only a fraction of the job so far.
+
+Please refrain from making requests for any planned or existing features from either ACE2, AGM or CSE. Most of them are already being or have been considered for porting or a rewrite.
+
+<div class="panel callout">
+    <h5>Note:</h5>
+    <p>Due to our current work load, in the period of weeks prior and posterior to the first release, new feature requests are under embargo. Feel free to submit yours now, but don't expect them to be considered just yet. Any treatment or attention by developers towards the feature request thread is highly unlikely during this time.</p>
```
Comment (Glowbal) [inline CONTRIBUTING.md]: This note does no longer apply.

**Comment by alganthe** [comment]
Updated, @bux links are at the bottom of the page under ( Regarding Pull Requests (PRs) )

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+Please refrain from making requests for any planned or existing features from either ACE2, AGM or CSE. Most of them are already being or have been considered for porting or a rewrite.
+
+## Requesting a feature
+In order to avoid duplicates and keep the issue tracker organized, we have created a common issue for <a href="{{ site.githubUrl }}/issues/414/" target="_blank">ACE 3 Feature requests</a>. Any and all relevant requests should be submitted there, where they will also get discussed and evaluated. Before adding a new one, make sure to check the previous entries from the thread and do a quick search for similar suggestions; please don't reiterate requests for features that had already been accepted for inclusion, or those which were disregarded earlier.
+
+Following their approval, feature requests may be moved by moderators to a separate issue for further discussion.
+
+# Regarding Pull Requests (PRs)
+- You want to help but don't know where to start ? check the wiki entry for [setting up the development environment](http://ace3mod.com/wiki/development/setting-up-the-development-environment.html)
```
Comment (bux578) [inline CONTRIBUTING.md]: `start ? check` -> `start? Check`

**Comment by gienkov** [comment]
Maybe tell ppl to use debug button instead of "placed ace3 modules"?

**Comment by alganthe** [comment]
line added.

**Pull request merged**